### PR TITLE
chore(gitignore): add few missing paths

### PR DESCRIPTION
**Pull request by mastilver**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,7 @@
 node_modules
+build
+
+*.log
+
+.jshintignore
```
Comment (ericelliott) [inline .gitignore]: These should not be ignored.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,7 @@
...
+.jshintrc
```
Comment (ericelliott) [inline .gitignore]: See above.

**Comment by mastilver** [comment]
So shouldn't it the included then?

**Comment by ericelliott** [comment]
They need to be part of the git repo, so they should not be in `.gitignore`.

**Comment by ericelliott** [comment]
Oh, I see. This project uses eslint. =)

**Pull request merged**